assets/js/blocks/attribute-filter/edit.js

```diff
@@ -0,0 +1,301 @@
+					'woo-gutenberg-products-block'
+				) }
+			</p>
+			<Button
+				className="wc-block-attribute-filter__add_attribute_button"
+				isDefault
+				isLarge
+				href={
+					ADMIN_URL +
+					'edit.php?post_type=product&page=product_attributes'
+				}
+			>
+				{ __( 'Add new attribute', 'woo-gutenberg-products-block' ) +
+					' ' }
+				<IconExternal />
+			</Button>
+			<Button
+				className="wc-block-attribute-filter__read_more_button"
+				isTertiary
+				href="https://docs.woocommerce.com/document/managing-product-taxonomies/"
+			>
+				{ __( 'Learn more', 'woo-gutenberg-products-block' ) }
+			</Button>
+		</Placeholder>
+	);
+
+	const onDone = useCallback( () => {
+		setIsEditing( false );
+		debouncedSpeak(
+			__(
+				'Showing attribute filter block preview.',
+				'woo-gutenberg-products-block'
+			)
+		);
+	}, [] );
+
+	const onChange = useCallback( ( selected ) => {
+		setAttributes( {
+			attributeId: selected[ 0 ].id,
+		} );
+	}, [] );
+
+	const renderAttributeControl = () => {
+		const { attributeId } = attributes;
+
+		const messages = {
+			clear: __(
+				'Clear selected attribute',
+				'woo-gutenberg-products-block'
+			),
+			list: __( 'Product Attributes', 'woo-gutenberg-products-block' ),
+			noItems: __(
+				"Your store doesn't have any product attributes.",
+				'woo-gutenberg-products-block'
+			),
+			search: __(
+				'Search for a product attribute:',
+				'woo-gutenberg-products-block'
+			),
+			selected: ( n ) =>
+				sprintf(
+					_n(
+						'%d attribute selected',
+						'%d attributes selected',
+						n,
+						'woo-gutenberg-products-block'
+					),
+					n
+				),
+			updated: __(
+				'Product attribute search results updated.',
+				'woo-gutenberg-products-block'
+			),
+		};
+
+		const list = sortBy(
+			toArray(
+				mapValues( ATTRIBUTES, ( item ) => {
+					return {
+						id: parseInt( item.attribute_id, 10 ),
+						name: item.attribute_label,
+					};
+				} )
+			),
+			'name'
+		);
+
+		return (
+			<SearchListControl
+				className="woocommerce-product-attributes"
+				list={ list }
+				selected={ list.filter( ( { id } ) => id === attributeId ) }
+				onChange={ onChange }
+				messages={ messages }
+				isSingle
+			/>
+		);
+	};
+
+	const renderEditMode = () => {
+		return (
+			<Placeholder
+				className="wc-block-attribute-filter"
+				icon={ <Gridicon icon="menus" /> }
+				label={ __(
+					'Filter Products by Attribute',
+					'woo-gutenberg-products-block'
+				) }
+				instructions={ __(
+					'Display a list of filters based on a chosen attribute.',
+					'woo-gutenberg-products-block'
+				) }
+			>
+				<div className="wc-block-attribute-filter__selection">
+					{ renderAttributeControl() }
+					<Button isDefault onClick={ onDone }>
+						{ __( 'Done', 'woo-gutenberg-products-block' ) }
+					</Button>
+				</div>
+			</Placeholder>
+		);
+	};
+
+	return (
+		<Fragment>
+			{ 0 === ATTRIBUTES.length ? (
```

Comment:
There is something weird here, `ATTRIBUTES` is an object with weird keys, for me, so this check is always false:
![image](https://user-images.githubusercontent.com/3616980/68315411-05d44000-00b8-11ea-839c-63b2bb101aeb.png)
Can you reproduce that too?